Library/Formula/libcutl.rb

```diff
@@ -0,0 +1,24 @@
+class Libcutl < Formula
+  homepage "http://www.codesynthesis.com/projects/libcutl/"
+  url "http://www.codesynthesis.com/download/libcutl/1.9/libcutl-1.9.0.tar.gz"
+  sha256 "1b575aa8ed74aa36adc0f755ae9859c6e48166a60779a5564dd21b8cb05afb7d"
+
+  depends_on "gcc" => :build
```

Comment:
The thing is that "odb" (which depends on libcutl) uses gcc plugins and needs to be built with gcc, so is libcutl.

If I use only `fail_with`, do I need to specify "every" compiler? Because the issue isn't really `clang`, it's the `gcc` requirement